hashicorp/boundary-ui
# chore: 🤖 Migrate `form.inut` to `Hds::TextInput::Field`

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-12477

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-12477)

## Description
- Migrate `form/create-worker-led` form to use `Hds:Text::InputField` instead of `form.input`
- Fix corresponding test



## Screenshots (if appropriate):
### Before:
![image](https://github.com/hashicorp/boundary-ui/assets/24277002/94549fe7-3d4d-4292-bbef-c9248e843bde)

### After: 
![image](https://github.com/hashicorp/boundary-ui/assets/24277002/d1db6a8e-e4a7-4cb5-8ff2-5546302676a3)


## How to Test
- Run this PR locally using a Boundary dev instance, start the admin ui using `ENABLE_MIRAGE=false API_HOST=http://localhost:9200 yarn start`, and authenticate
- Click on "Workers in sidebar navigation" and select the "New" button
- Click on "Register Worker" to ensure that error messaging fails the submit for the `worker _generated_auth_token`
- Follow these instructions from the test plan for [BYOW](https://docs.google.com/spreadsheets/d/1IOwd54cyUqp1G51amSi_5EPwsxsaprXKTqiH5-z-dc0/edit#gid=109027861) to successfully create a `worker`
- Ensure that after clicking "Register Worker" the button at the bottom of the page converts to "Done"
- Ensure that the worker is listed in the workers list page.





:technologist: [Admin preview](PATH_TO_FEATURE)

:desktop_computer: [Desktop preview](PATH_TO_FEATURE)

## Checklist:


~~- [ ] I have added JSON response output for API changes~~
~~- [ ] I have added steps to reproduce and test for bug fixes in the description~~
~~- [ ] I have commented on my code, particularly in hard-to-understand areas~~
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~
